presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -2776,6 +2776,210 @@ public void testCollectColumnStatisticsOnInsert()
         assertUpdate(format("DROP TABLE %s", tableName));
     }
 
+    @Test
```

Comment:
This test looks good, but could you please update the product test (TestHiveTableStatistics) as well? I think since we got our own analyze we no longer need to call Hive's analyze . So you can simply replace the database the ANALYZE is being called at. 

CC: @findepi @kokosing

Reply:
@jessesleeping Let me know if you need any help with figuring out product tests

Reply:
@jessesleeping @arhimondr i am concerned about removing analyze-on-hive from product tests. We test with different Hive versions and they can be setting different table/partition properties during ANALYZE (now or some future versions, like Hive 3). We should have tests ensuring proper interop between Hive's ANALYZE and Presto's reading statistics. 

However, as we have our ANALYZE now, we could try the contrary -- but I don't know if Hive has useful SHOW STATS (or equivalent)

Reply:
We are not testing Hive's analyze. We are testing that Presto can handle table properties as those set by Hive's analyze.

What about defining some 3 tables (unanalyzed, analyzed, analyzed with columns) directly in the image and analyze it during image build time?
Of course, having test data setup in the image is not elastic (eg think we need to cover new data type), but maybe saved time pays off overall.
or, maybe, we can speed up analyze in Hive by using different execution framework?